### PR TITLE
Adds hotfixes section to RELEASING doc

### PR DESCRIPTION
**Pull request by vegaro**
Added a section to clarify how a hotfix should be done

### CODE REVIEW

File: Contributing/RELEASING.md [hunk 1]
```diff
@@ -7,3 +7,13 @@
...
 4. The rest will be performed automatically by CircleCI. If the automation fails, you can revert to manually calling `bundle exec fastlane deploy`.
 5. The release branch PR can be merged after the release is completed.
+
+##### Hotfixes:
+Sometimes you might need to release a patch on a version that's not the latest. Or sometimes there might have been commits on `main` that shouldn't be released, but the latest version has a big bug. For example, let's say the last version is 4.0.0, but there's a bug on 3.9.0 that needs to be released as a 3.9.1:
+1. Open a PR with the fix, merge it to `main`.
+1. Jump to 3.9.0 and create a new branch `release/3.9.0`. Push this branch since it will be used as base for the PR you will open in the next steps.
+1. Run `bundle exec fastlane ios bump` as you would do with any release. This will create a new PR `release/3.9.1`. Make sure the base branch of the PR is pointing to `release/3.9.0`.
+1. Cherry pick the changes with your fix that you just merged to `main`. Fix any conflicts if there are any.
```
Comment (joshdholtz) [inline Contributing/RELEASING.md:16]: Can we give an example of the cherry pick command here for anybody not familiar with it?

File: Contributing/RELEASING.md [hunk 2]
```diff
@@ -7,3 +7,13 @@
...
+1. When the PR is approved, **DO NOT MERGE IT**. Approve the hold job in CircleCI as you would do for any other release. If there is no hold job because the release is older than when we introduced that job, manually tag the last commit in `release/3.9.1` with `3.9.1`.
```
Comment (joshdholtz) [inline Contributing/RELEASING.md:17]: Can we say to put the "DO NOT MERGE" label on the PR?

**Review by joshdholtz — APPROVED** [review]
Few small thoughts otherwise 🔥

File: Contributing/RELEASING.md (hunk 2 above)
Comment (vegaro) [reply Contributing/RELEASING.md:17]: Great idea

**Comment by vegaro** [comment]
We can automate most of this in a special `hotfix_release` lane but for now this is good I think

**Review by NachoSoto — APPROVED** [review]

**Pull request merged**